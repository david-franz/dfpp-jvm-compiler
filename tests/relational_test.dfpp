module tests.rel

fn main() {
  print('lt? ' + (1 < 2))
  print('ge? ' + (3 >= 3))
  print('eq? ' + (4 == 5))
}
