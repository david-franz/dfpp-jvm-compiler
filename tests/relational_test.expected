lt? true
ge? true
eq? false
